source/src/main/java/org/cerberus/core/service/sql/ISQLService.java

```diff
@@ -86,9 +86,20 @@ public interface ISQLService {
      * @param system
      * @param columnsToGet
      * @param columnsToHide
+	 * @param ignoreNoMatchColumns         if <code>true</code> then ignore error if
+	 *                                     at least one column to get does not match
+	 *                                     with the SQL query result. In that case,
+	 *                                     each no match column's value will be set
+	 *                                     to the {@link String} empty value ("").
+	 *                                     Use {@link #IGNORE_NON_MATCHED_COLUMNS} &
+	 *                                     {@link #DO_NOT_IGNORE_NON_MATCHED_COLUMNS}
+	 *                                     for better readability
+	 * @param defaultNonMatchedColumnValue the default value to set to any
+	 *                                     non-matched column if necessary
      * @param execution
      * @return
      */
-    AnswerList<HashMap<String, String>> queryDatabaseNColumns(String connectionName, String sql, int rowLimit, int defaultTimeOut, String system, HashMap<String, String> columnsToGet, List<String> columnsToHide, TestCaseExecution execution);
-
+    AnswerList<HashMap<String, String>> queryDatabaseNColumns(String connectionName, String sql, int rowLimit, int defaultTimeOut, String system, HashMap<String, String> columnsToGet, List<String> columnsToHide, boolean ignoreNoMatchColumns, String defaultNonMatchedColumnValue, TestCaseExecution execution);
+    boolean IGNORE_NON_MATCHED_COLUMNS = true;
```

Comment:
Here as well.

Reply:
These variables were used for better readability. The latest commit (#17ba4e0) removes them.